chainguard-dev/edu
# Cloudsmith pull through doc

## Type of change

This PR adds a new pull-through guide focused on Cloudsmith. It walks readers through setting up a Cloudsmith repo and upstream proxy they can use as a pull through cache for both Developer and Production Images.

I also made some small tweaks to our existing pull through guides.

### What should this PR do?

resolves https://github.com/chainguard-dev/internal/issues/4047

### Why are we making this change?


### What are the acceptance criteria? 


n/a

### How should this PR be tested?

This may be slightly annoying to test, as you'll need to set up a Cloudsmith account and start a free trial. Should have a full tech test and a close read through for spelling/grammar/style.
